filters/apiusagemonitoring/pathinfo.go

```diff
@@ -46,13 +59,19 @@ func (s pathInfoByRegExRev) Len() int           { return len(s) }
 func (s pathInfoByRegExRev) Less(i, j int) bool { return s[i].Matcher.String() > s[j].Matcher.String() }
 func (s pathInfoByRegExRev) Swap(i, j int)      { s[i], s[j] = s[j], s[i] }
 
-type metricNames struct {
-	GlobalPrefix            string
+type endpointMetricNames struct {
+	EndpointPrefix          string
 	CountAll                string
-	CountPerStatusCodeRange [5]string
+	CountPerStatusCodeRange [6]string
```

Comment:
If you would use slices instead of arrays you would not need to change it.

Reply:
But for something constant size, is it not better to use _arrays_ instead of _slices_? It thought it would be one of the "basic" optimisation in _GO_.

Reply:
to me this rather feels like microoptimization. Typically, I use arrays only when I can measure the difference. But I also don't have a strong opinion on this.